### ISSUE TITLE
reproduce the result

### ISSUE THREAD

**Issue by Backdrop9019** [kahnchana/svt#5]
Hi  Thank you for nice paper and code.
I'm trying to reproduce a ucf101 dataset (linear 90% acc).
I used the same parameter as the paper, but it didn't work :(
Can you share the script for each experiment result?

Thanx

**Comment by kahnchana** [comment]
Hi, thank you for the positive comments. Did you run into a collapse during the ssl training part? What values did you get? 

Yes, I am trying to setup a quickstart with exact scripts to directly run each evaluation. Will do this soon. All the params we use are the ones mentioned in the paper.

**Comment by Backdrop9019** [comment]
Thank you for your prompt reply.
I use kinetics400 pre-train weight  and learn linear ucf101 with learning rate 8e-3. 
I used the same parameters as the paper, but i get  "test_acc1": 89.2413. (In the paper, it
is 90.8%)

**Comment by kahnchana** [comment]
Do you use slow-fast inference for the linear eval? I just realized I have not been able to update `eval_linear.py` with that. I am working on updating that soon.

**Comment by Backdrop9019** [comment]
Oh, I haven't used it. Thank you for your feedback, I'll wait for the update.
